client/components/layout/search.tsx

```diff
@@ -0,0 +1,394 @@
+            "link": "https://api.spotify.com/v1/artists/0LcJLqbBmaGUft1e9Mm8HV",
+            "uri": "spotify:artist:0LcJLqbBmaGUft1e9Mm8HV",
+            "icon_link": "https://i.scdn.co/image/ab6761610000e5eb124eba6bf3476404531bd7b2"
+        },
+        {
+            "name": "Amaranthe",
+            "link": "https://api.spotify.com/v1/artists/2KaW48xlLnXC2v8tvyhWsa",
+            "uri": "spotify:artist:2KaW48xlLnXC2v8tvyhWsa",
+            "icon_link": "https://i.scdn.co/image/ab6761610000e5eb293f7f4aebb31292f607bf3b"
+        },
+        {
+            "name": "Apulanta",
+            "link": "https://api.spotify.com/v1/artists/5kwthnxNdfnqGk0nL35wDC",
+            "uri": "spotify:artist:5kwthnxNdfnqGk0nL35wDC",
+            "icon_link": "https://i.scdn.co/image/ab6761610000e5eb9c726630964d9ca3cbb56d59"
+        },
+        {
+            "name": "Apocalyptica",
+            "link": "https://api.spotify.com/v1/artists/4Lm0pUvmisUHMdoky5ch2I",
+            "uri": "spotify:artist:4Lm0pUvmisUHMdoky5ch2I",
+            "icon_link": "https://i.scdn.co/image/ab6761610000e5ebd5b5240aa1e0454e3ea2eb72"
+        },
+    ]
+
+    const playlist: Playlist[] = [
+        {
+            "name": "AC/DC – Greatest Hits",
+            "link": "https://api.spotify.com/v1/playlists/0wTrfNrOLJnH8XJed32zaE",
+            "uri": "spotify:playlist:0wTrfNrOLJnH8XJed32zaE",
+            "icon_link": "https://image-cdn-ak.spotifycdn.com/image/ab67706c0000bebb580ae2312e0f2751cad23b24"
+        },
+        {
+            "name": "ABBA: Best Of The Best",
+            "link": "https://api.spotify.com/v1/playlists/1uNj91vKo7JPMaTP6CpU6F",
+            "uri": "spotify:playlist:1uNj91vKo7JPMaTP6CpU6F",
+            "icon_link": "https://image-cdn-ak.spotifycdn.com/image/ab67706c0000bebbd3e8b10c2b93e3b179c88c1a"
+        },
+        {
+            "name": "ABBA Gold: Greatest Hits",
+            "link": "https://api.spotify.com/v1/playlists/5FAJg17Von0Mc8jkHUJQDf",
+            "uri": "spotify:playlist:5FAJg17Von0Mc8jkHUJQDf",
+            "icon_link": "https://image-cdn-ak.spotifycdn.com/image/ab67706c0000bebb932bd2a9feb1bba5ff292382"
+        },
+        {
+            "name": "This Is ABBA",
+            "link": "https://api.spotify.com/v1/playlists/37i9dQZF1DZ06evO0nVzAk",
+            "uri": "spotify:playlist:37i9dQZF1DZ06evO0nVzAk",
+            "icon_link": "https://thisis-images.spotifycdn.com/37i9dQZF1DZ06evO0nVzAk-large.jpg"
+        },
+        {
+            "name": "amispoppia😍",
+            "link": "https://api.spotify.com/v1/playlists/6MIGmlQ5sTBHD0b4A71rzU",
+            "uri": "spotify:playlist:6MIGmlQ5sTBHD0b4A71rzU",
+            "icon_link": "https://image-cdn-ak.spotifycdn.com/image/ab67706c0000bebb219af830eb68824d4b9df2a2"
+        }
+    ]
+
+    useEffect(() => {
+        setTrackList(track)
+        setAlbumList(album)
+        setArtistList(artist)
+        setPlaylistList(playlist)
+    }, [])
+    // ------------------- For local development
```

Comment:
I don't think this should be in main. Just use the server locally or if this is absolutely necessary then store it somewhere else.

Reply:
Hups